_notebooks/2020-03-27-covid19-progress-projections.ipynb

```diff
@@ -0,0 +1,2122 @@
+{
```

Comment:
<p>This question/answer format is confusing.  Give a very brief description in two sentences of what this dashboard is about.</p><p> </p><p>I don't understand what this statement is saying</p><blockquote>`Note: Assumption of current infection rates is not to say that they won't be reduced, but to understand the implications of not reducing them.</blockquote><p> </p><p>Consider deleting this and putting the general disclaimer we have on all dashboards with predictive models for consistency example:  <a href="https://covid19dashboards.com/covid-19-us-case-estimation/#Definition-Of-Fields:" target="_blank">https://covid19dashboards.com/covid-19-us-case-estimation/#Definition-Of-Fields:</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/github/covid19-dashboard/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Changed completely, hope it helps.</p><p> </p>